pipeline_dp/combiners.py

```diff
@@ -9,6 +14,18 @@ class Combiner(abc.ABC):
     aggregation state. Combiners contain logic, while accumulators contain data.
     The API of combiners are inspired by Apache Beam CombineFn class.
     https://beam.apache.org/documentation/transforms/python/aggregation/combineperkey/#example-5-combining-with-a-combinefn
+
+    Let we have some dataset X to aggregate. The workflow of running an
```

Comment:
(nit) Here's how PipelineDP uses combiners to performs an aggregation on some dataset X:
1. ....

Reply:
Thanks!